content/en/docs/concepts/policy/limit-range.md

```diff
@@ -377,7 +373,7 @@ kubectl delete ns limitrange-demo
 
 ## Example
 
-See  [a tutorial on how to limit compute resources per Container/Pod](/docs/tasks/administer-cluster/manage-resources/cpu-constraint-namespace/) .
+See  [a tutorial on how to limit compute resources per namespace](/docs/tasks/administer-cluster/manage-resources/cpu-constraint-namespace/) .

```

Comment:
nit: Examples
nit: Make a list of examples

Reply:
Done thanks